recipe/meta.yaml

```diff
@@ -1,21 +1,20 @@
-{% set version = "7.0.5" %}
-{% set sha256 = "faf8dd3bfa4a3b0036e8b62a5f647a0596bfd579f31c6cc00b2ce2f82bcf9de1" %}
+{% set version = "7.0.6.1" %}
 
 package:
   name: epics-base
   # Always use 4 numbers in the version to match run_exports!
-  version: {{ version }}.0
+  version: {{ version }}
```

Comment:
We need to remember adding the `.0` again.
If we don't, it won't fail but will produce a recipe where the max pin is `7.0.7`, meaning that it will be seen as ABI compatible with `7.0.7.1` when it's released, which is probably not the case... I saw that issue in the past.

I wish epics-base versions were always using 4 numbers.

Maybe, it would be more obvious to put this comment on the first line, and hard-code the version in the url instead?
I agree it's easy to miss otherwise.

Reply:
I updated the recipe. I hope it's more explicit like that.

Reply:
Thanks!